examples/blog-multiple-authors/src/pages/authors/[author].astro

```diff
@@ -11,8 +11,16 @@ let canonicalURL = Astro.request.canonicalURL;
 
 // collection
 import authorData from '../../data/authors.json';
+
+interface Frontmatter {
```

Comment:
(same comment for other `Frontmatter` interfaces across the rest of the PR) Frontmatter is a little vague since we sometimes/often call the component script the "component frontmatter". Can this be `MarkdownFrontmatter` instead?